src/Codeception/Lib/DbPopulator.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Codeception\Lib;
+
+/**
+ * Populates a db using a parameterized command built from the Db module configuration.
+ */
+class DbPopulator
+{
+    /**
+     * The command to be executed.
+     *
+     * @var string
+     */
+    private $builtCommand;
+
+    /**
+     * @var array
+     */
+    protected $config;
+
+    /**
+     * Constructs a DbPopulator object for the given command and Db module.
+     *
+     * @param $config
+     * @internal param string $command The parameterized command to evaluate and execute later.
+     * @internal param Codeception\Module\Db|null $dbModule The Db module used to build the populator command or null.
+     */
+    public function __construct($config)
+    {
+        $this->config = $config;
+        $command = $this->config['populator'];
+        $this->builtCommand = $this->buildCommand((string) $command);
+    }
+
+    /**
+     * Builds out a command replacing any found `$key` with its value if found in the given configuration.
+     *
+     * Process any $key found in the configuration array as a key of the array and replaces it with
+     * the found value for the key. Example:
+     *
+     * ```php
+     * <?php
+     *
+     * $command = 'Hello $name';
+     * $config = ['name' => 'Mauro'];
+     *
+     * // With the above parameters it will return `'Hello Mauro'`.
+     * ```
+     *
+     * @param string $command The command to be evaluated using the given config
+     * @param array $config The configuration values used to replace any found $keys with values from this array.
+     * @return string The resulting command string after evaluating any configuration's key
+     */
+    protected function buildCommand($command)
```

Comment:
@tiger-seo yes, get and set are either for property values or just to retrieve something where no computation happens. At least that's the implicit "logic" that I would think a method has.